lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -306,6 +306,13 @@ findDatabases tr dir = do
   where
     expectedPrefix = T.pack $ keyTypeDescriptor $ Proxy @k
 
+-- | A data-type for capturing column status. Used to be represented as a
+-- 'Maybe Bool' which is somewhat confusing to interpret.
+data SQLColumnStatus
```

Comment:
Suggestion: use PascalCase here, treating SQL as a word.

```suggestion
data SqlColumnStatus
```

Similar examples:
- [SqlBackend](https://hackage.haskell.org/package/persistent-2.10.5.1/docs/Database-Persist-Sql.html#t:SqlBackend)
- [InsertSqlResult](https://hackage.haskell.org/package/persistent-2.10.5.1/docs/Database-Persist-Sql.html#t:InsertSqlResult)

Reply:
I do actually prefer the golang guidelines on acronyms, but whatever ^.^